### PR TITLE
Collection

### PR DESCRIPTION
**Pull request by SADSASD1**
New scanner

### CODE REVIEW

File: Main.java [hunk 1]
```diff
@@ -1,27 +1,53 @@
...
+            System.out.println("Your firstname is" + " " + names);
+            System.out.println("Your lastname is" + " " + lastnames);
+            System.out.println("Your email is" + " " + emails);
+            System.out.println("Your email is" + " " + passwords);
+
+
+            Scanner in = new Scanner(System.in);
+            System.out.println("Do you want to create another user");
+            answer = in.next();
+
```
Comment (tigran10) [inline Main.java:40]: Ba user ches sarqum ?

et vor havaqumes `name`, `lastname`, `password`, ev ayln - dranic heto porci `User` sarqes et tvyalnerov, u havaqes ArrayList-i mej

Comment (SADSASD1) [reply Main.java:40]: Tigran jan barev    
Tigran jan chem karum en masy erb kody run em talis usery tvaylnery lracnum em heto noric em user sarqum u et sax vor lcnuma Arrayi mej chi cuyc talis:
inch kod grem vor verjum sax userneri tvyalnery cuyc ta

Comment (tigran10) [reply Main.java:40]: et kody gone push ara inch vor mi branch vor naem vor masy chi stacvum

Comment (tigran10) [reply Main.java:40]: pti senc mi ban lini

```

List<User> users = new Arraylist<>;

// Scanner create new users

for (User user : users){
            System.out.println(user);
  }
```

**Pull request merged**